packages/vite/src/node/plugins/asset.ts

```diff
@@ -250,7 +250,7 @@ async function fileToBuiltUrl(
   return url
 }
 
-function getAssetHash(content: Buffer) {
+export function getAssetHash(content: Buffer) {
```

Comment:
Please add the return type typedef
This is needed when exporting a function, so the code that uses this function becomes stable